pkgs/development/python-modules/artifacts/default.nix

```
@@ -0,0 +1,23 @@
+{ buildPythonPackage, fetchPypi, lib, pyyaml }:
+
+buildPythonPackage rec {
+  pname = "artifacts";
+
+  version = "20221219";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-rCmmzA9OmUiJPVrOJu2N61zj6Z4MysJP6fOTKRibCm0=";
+  };
+
+  propagatedBuildInputs = [ pyyaml ];
```

Was changed recently.

```suggestion
  dependencies = [ pyyaml ];
```